Makefile

```diff
@@ -201,18 +201,15 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 		-t ../../../Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md \
 		-t ../../../Documentation/Helm-Charts/_templates.gotmpl
 
-check-helm-docs:
+check.helm-docs:
```

Comment:
```
all                  check-helm-docs      docs-build           go.fmt               go.test.unit         mod.check
build                clean                docs-preview         go.init              go.validate          mod.update
build.all            codegen              fmt                  go.install           go.vet               prune
build.common         crds                 gen-csv              go.lint              helm.build           test
build.version        csv-clean            generate-docs-crds   go.mod.check         helm-docs            test-integration
bundle               distclean            gen-rbac             go.mod.clean         help                 vet
check                do.build.parallel    go.build             go.mod.update        install              
check-docs           docs                 go.check             go.test.integration  lint                 
```

We have a lot on unconsistent behaviour is there a norm we should opt and do update everything in a consistent way

Reply:
@parth-gr wrote:
> ```
> all                  check-helm-docs      docs-build           go.fmt               go.test.unit         mod.check
> build                clean                docs-preview         go.init              go.validate          mod.update
> build.all            codegen              fmt                  go.install           go.vet               prune
> build.common         crds                 gen-csv              go.lint              helm.build           test
> build.version        csv-clean            generate-docs-crds   go.mod.check         helm-docs            test-integration
> bundle               distclean            gen-rbac             go.mod.clean         help                 vet
> check                do.build.parallel    go.build             go.mod.update        install              
> check-docs           docs                 go.check             go.test.integration  lint                 
> ```
> 
> We have a lot on unconsistent behaviour is there a norm we should opt and do update everything in a consistent way

I am not aware of any general norm gere but was simply trying to add some consistency by introducing or rather extening a `check.foo`pattern.

Reply:
Okay,